src/connectedvmware/azext_connectedvmware/tests/latest/test_connectedvmware_scenario.py

```diff
@@ -165,6 +165,9 @@ def test_connectedvmware(self):
         # At least 1 disk should be there for the vm resource.
         assert len(resource_list) >= 1
 
+        # Update VM.
+        self.cmd('az connectedvmware vm update -g {rg} --name {vm_name} --memory-size 2048 --num-CPUs 2')
```

Comment:
Could you please add check for this test case?

Reply:
Added check

Reply:
Sorry, I don't see any new commits. Did you push the new code to the remote branch?

Reply:
Sorry the last push was got rejected, again pushes the change.